Update existing evaluation pages (#1180)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/evaluation_on_gadi/ilamb_workflow.md

## What is ILAMB?

> The [International Land Model Benchmarking (_ILAMB_)](https://www.ilamb.org/) benchmarking system is a python framework used to quantitatively compare a defined set of observable variables with a number of land models.

Reviewer: ```suggestion
The [International Land Model Benchmarking (_ILAMB_)](https://www.ilamb.org/) benchmarking system is a Python framework used to quantitatively compare a defined set of observable variables with a number of land models.
```

> This documentation is tailored to using _ILAMB_ on _Gadi_ and, hence, it supplements rather than replaces the official documentation. Users are encouraged to read the  [ILAMB documentation](https://www.ilamb.org/doc/) and relevant [tutorials](https://www.ilamb.org/doc/tutorial.html).

Reviewer: ```suggestion
This documentation is tailored to using _ILAMB_ on _Gadi_ and, hence, it supplements _ILAMB_'s official documentation rather than replacing it. Users are encouraged to read the [ILAMB official documentation](https://www.ilamb.org/doc/) and relevant [tutorials](https://www.ilamb.org/doc/tutorial.html).
```

!!! note

>     ACCESS-NRI is supporting an NCI configuration of ILAMB under the name _ILAMB-workflow_ via the [_conda/analysis3_](/getting_started/environments) environment.

Reviewer: ```suggestion
    ACCESS-NRI is supporting an NCI configuration of _ILAMB_ under the name _ILAMB-workflow_ via the [_conda/analysis3_](/getting_started/environments) environment.
```

> *ILAMB-workflow* is the ACCESS-NRI software and data infrastructure that enables the ILAMB evaluation framework on NCI Gadi. It includes the 

Reviewer: ```suggestion
*ILAMB-workflow* is the ACCESS-NRI software and data infrastructure that enables the _ILAMB_ evaluation framework on NCI _Gadi_. It includes:
```

### Pre-requisites

To use *ILAMB* on *Gadi* ensure you fulfill the [Set Up your NCI Account](/getting_started/set_up_nci_account) section.

> _ILAMB_ is provided through the [xp65](https://my.nci.org.au/mancini/project/xp65/join) NCI projects on _Gadi_, so you need to have an NCI account and be a member of this projects to use it. 

Reviewer: Same as suggested above for ESMValTool, please copy the same text/structure.

Depending on your needs, you may want to also join the following supported data collections:

- CMIP6: [fs38](https://my.nci.org.au/mancini/project/fs38/join), [oi10](https://my.nci.org.au/mancini/project/oi10/join)
- CMIP5: [rr3](https://my.nci.org.au/mancini/project/rr3/join), [al33](https://my.nci.org.au/mancini/project/al33/join)

> ### Loading the ILAMB-workflow modules

Reviewer: ```suggestion
### Enable ILAMB-workflow commands
```

> To load the the *ilamb* module, execute the following commands:

Reviewer: There is no "ilamb" module.

```suggestion
To enable the _ILAMB-workflow_ command, load the the `conda/analysis3` module by running:
```

```
    module use /g/data/xp65/public/modules
    module load conda/analysis3
```

> Visit [ACCESS-NRI documentation](https://ilamb-workflow.readthedocs.io/en/latest/?badge=latest) on how to run _ILAMB_ on _Gadi_.

Reviewer: ```suggestion
Visit [ILAMB-workflow documentation](https://ilamb-workflow.readthedocs.io/en/latest/?badge=latest) for further information on how to use _ILAMB_ on _Gadi_.
```

This should also be in the paragraph about "How to use ILAMB-workflow" mentioned [here](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1180/changes#r3245981504)

> To run _ILAMB_, you need to execute the command `ilamb-run` with a number of arguments/ files:

Reviewer: The content from this line below should not be in the same paragraph as the "enable ILAMB".

I would place it in a separate paragraph called something like "How to use ILAMB-workflow"

```
ilamb-run --config config.cfg --model_setup model_setup.txt --regions regions.txt
```

    * _config.cfg_ defines which observables and observational datasets to be compared.
    * _model_setup.txt_ defines the paths of models that to be compared.
    * _regions.txt_ defines the regions to be compared, e.g., _global_, _aust_ (Australia), etc.

While these files can be self-defined, ACCESS-NRI provides the necessary files and tools to set your model paths to run on _Gadi_. All you need to do is decide which observations and models you wish to compare. 

NCI hosts replicas of the _ILAMB_ observational data sets through the NCI project [ct11](https://my.nci.org.au/mancini/project/ct11/join) as well as a large amount of model outputs are available on _Gadi_, such as ACCESS model output.

For more information, refer to [Finding ACCESS Data](/model_evaluation/data/finding) on how to find data on NCI.

To learn more about how to adjust the _ILAMB_ setup, refer to the official [ILAMB documentation](https://www.ilamb.org/doc/) and relevant [tutorials](https://www.ilamb.org/doc/tutorial.html).

> ## Example: CMIP6 comparisons and ACCESS ESM1.5 benchmarking

Reviewer: This could be a sub-paragraph of the "How to use ILAMB-workflow" paragraph mentioned [here](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1180/changes#r3245981504).

> ACCESS-NRI is maintaining a collection of benchmark comparisons for the ACCESS community, such as that with Coupled Model Intercomparison Project (CMIP) data, see in the [workflow documentation.](https://ilamb-workflow.readthedocs.io/en/latest/source/ILAMB.html#ilamb-cmip-confrontations-maintained-by-access-nri)

Reviewer: ```suggestion
ACCESS-NRI maintains a collection of benchmark comparisons for the ACCESS community, including comparisons with data from CMIP. See the [workflow documentation](https://ilamb-workflow.readthedocs.io/en/latest/source/ILAMB.html#ilamb-cmip-confrontations-maintained-by-access-nri) for details.
```

> In the following example, the supported ACCESS Earth System Model (ESM) [ACCESS-ESM1.5](/models/access_models/access-esm#access-esm1.5) is compared with two other ESM models:

Reviewer: ```suggestion
In the following example, [ACCESS-ESM1.5](/models/access_models/access-esm#access-esm1.5) is compared with two other ESM models:
```

- [BCC ESM1 (Beijing Climate Center Earth System Model version 1)](https://gmd.copernicus.org/articles/13/977/2020/)
- [CanESM5 (Canadian Earth System Model version 5)](https://gmd.copernicus.org/articles/12/4823/2019/gmd-12-4823-2019.html)

Numerous benchmark comparisons have been defined in the configuration file. The comparison of variables have been organised under different sections, such as the _Hydrology Cycle_. 

> For other variables, such as the _Gross Primary Productivity_ (*gpp*), one or more datasets are available. For example, the gross primary productivity measurements of [FLUXNET2015](https://fluxnet.org/data/fluxnet2015-dataset/). 

Reviewer: ```suggestion
For other variables, such as the _Gross Primary Productivity_ (*gpp*), one or more datasets are available (e.g. the gross primary productivity measurements of [FLUXNET2015](https://fluxnet.org/data/fluxnet2015-dataset/)). 
```
